recipe/meta.yaml

```diff
@@ -35,10 +35,11 @@ test:
 
 about:
   home: https://github.com/tqdm/tqdm
-  license: Mozilla Public License 2.0 (MPL 2.0) or MIT License
+  license: MPLv2.0, MIT
   summary: 'A Fast, Extensible Progress Meter'
 
 extra:
   recipe-maintainers:
+    - casperdcl
```

Comment:
but can you add the tqdm org as a maintainer of conda-forge/tqdm-feedstock directly in github settings for this repo?